### ISSUE TITLE
Mask R-CNN: get all the parts and train specific ones

### ISSUE THREAD

**Issue by FiReTiTi** [pytorch/vision#2393]
Hi,
I would like to access all the different parts of Mask R-CNN in order to only train some of them.
I learnt in the discussion forum that I can use `requires_grad` to enable/disable training, but how can I access all the `trainable` parts?
Thanks,

**Comment by pmeier** [comment]
HI @FiReTiTi, please note that this issue tracker is not a help form and this issue will be closed. Our primary means of support is our discussion forum. 

---

You can access all parameters with `nn.Module.parameters()`. For the freezing part, have a look at [this question](https://discuss.pytorch.org/t/best-practice-for-freezing-layers/58156).

**Comment by FiReTiTi** [comment]
sorry.

**Comment by fmassa** [comment]
Closing following @pmeier answer.
